README.md

```diff
@@ -44,6 +44,7 @@ It doesn't require any modification to your application's code and let's you col
     ```ini
     extension=apm.so
     ```
+    Note: APM depends on JSON, so the apm.so extension must be loaded **after** JSON!
```

Comment:
Well, for some reason, it's still broken on PHP 5.6. Since moving the order fixed it for multiple systems, it is a good practise to order it this way, regardless of whether you have a patched PHP or not (because it will always work vs. might work depending on your distro). 

I usually prefer "works always" over "might work depending on factor X" :)